graalpy/graalpy-openai-starter/src/main/java/com/example/App.java

```diff
@@ -51,15 +52,18 @@ def create_chat_completion(user_input):
         }
     }
 
+    private static final TypeLiteral<Function<String, ChatCompletion>> STRING_TO_CHAT_COMPLETION_FUNCTION = new TypeLiteral<>() {
```

Comment:
Have you considered using 
```
 @FunctionalInterface
    public interface CompletionFunction {
        
        ChatCompletion apply(String choice);
        
    }
```

instead of TypeLiteral?

Reply:
even better!